### PR TITLE
Adjusting cmsweb and exporters Dockerfile

### PR DESCRIPTION
**Pull request by muhammadimranfarooqi**

### CODE REVIEW

**Comment by muhammadimranfarooqi** [comment]
@vkuznet 

Could you please review. Thanks

**Comment by muhammadimranfarooqi** [comment]
I am not sure about gopath. In new cmsweb image, I have following environment

```
PATH=/usr/sue/sbin:/usr/sue/bin:/usr/hdp/spark/bin:/data/cmsweb/bin:/data/cmsweb/gopath/bin:/data/cmsweb/gopath/src/github.com/vkuznet/cmsweb-exporters:/data/cmsweb/gopath/src/github.com/vkuznet/cmsweb-exporters/bin:/data/cmsweb/go/bin:/data/cmsweb:/data/cmsweb/go/bin:/data/cmsweb:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/hdp/hadoop/bin:/usr/hdp/hbase/bin
PWD=/data/cmsweb
JAVA_HOME=/usr/lib/jvm/jre-openjdk
LANG=en_US.UTF-8
SHLVL=1
HOME=/root
GOROOT=/data/cmsweb/go
LESSOPEN=||/usr/bin/lesspipe.sh %s
GOPATH=/data/cmsweb/gopath
OLDPWD=/data/cmsweb
_=/usr/bin/printenv
```

However, this directory is empty `GOPATH=/data/cmsweb/gopath`. Do we need this path?

File: docker/cmsweb/Dockerfile [hunk 1]
```diff
@@ -58,19 +62,19 @@ RUN mkdir -p $WDIR/bin
...
+#RUN cd cmsweb-exporters && \
+#    go build -o $WDIR/bin/das2go_exporter -ldflags="-s -w -extldflags -static" das2go_exporter.go && \
+#    go build -o $WDIR/bin/reqmgr_exporter -ldflags="-s -w -extldflags -static" reqmgr_exporter.go && \
+#    go build -o $WDIR/bin/wmcore_exporter -ldflags="-s -w -extldflags -static" wmcore_exporter.go && \
+#    go build -o $WDIR/bin/http_exporter -ldflags="-s -w -extldflags -static" http_exporter.go && \
+#    go build -o $WDIR/bin/process_exporter -ldflags="-s -w -extldflags -static" process_exporter.go && \
+#    go build -o $WDIR/bin/cpy_exporter -ldflags="-s -w -extldflags -static" cpy_exporter.go && \
+#    go build -o $WDIR/bin/cmsweb-ping -ldflags="-s -w -extldflags -static" cmsweb-ping.go
 
 ENV PATH="${GOPATH}/src/github.com/vkuznet/cmsweb-exporters:${GOPATH}/src/github.com/vkuznet/cmsweb-exporters/bin:${PATH}"
```
Comment (vkuznet) [inline docker/cmsweb/Dockerfile:79]: you don't need this anymore

**Comment by vkuznet** [comment]
you don't need anymore GOPATH

File: docker/cmsweb/Dockerfile (hunk 1 above)
Comment (muhammadimranfarooqi) [reply docker/cmsweb/Dockerfile]: ok. I have commented all relevant lines for `GOPATH`. Thanks

**Review by vkuznet — CHANGES_REQUESTED** [review]
Now, everything seems fine, just remove commented lines

**Comment by muhammadimranfarooqi** [comment]
done

**Pull request merged**